lib/inspec/resource.rb

```diff
@@ -122,6 +122,7 @@ def self.validate_resource_dsl_version!(version)
 require 'resources/packages'
 require 'resources/parse_config'
 require 'resources/passwd'
+require 'resources/pg_ident_conf'
```

Comment:
This is global feedback for this whole PR, putting it here since it made sense...

I think the resource name should be `postgres_ident_conf`... we should avoid abbreviations wherever possible, and we already have other PostgreSQL-related resources that are prefixed with `postgres_`